File: app/src/main/java/com/sky/chowder/ui/activity/PullDownActivity.java
package com.sky.chowder.ui.activity;

import android.os.Bundle;

import com.sky.base.BaseNoPActivity;
import com.sky.chowder.R;

public class PullDownActivity extends BaseNoPActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_pulldown);
    }

    @Override
    protected int getLayoutResId() {
        return R.layout.activity_pulldown;
    }

    @Override
    protected void initialize() {
        super.initialize();
        showToast(getIntent().getStringExtra("message"));
    }
}

File: app/src/main/java/com/sky/chowder/ui/activity/PullDownActivity.kt
package com.sky.chowder.ui.activity

import android.os.Bundle
import com.sky.chowder.R
import common.base.BaseNoPActivity

/**
 * Created by SKY on 2017/3/6.
 * 下拉动画
 */
class PullDownActivity : BaseNoPActivity() {

    override fun getLayoutResId(): Int = R.layout.activity_pulldown

    override fun initialize(savedInstanceState: Bundle?) = showToast(intent.getStringExtra("message") ?: "")
    override fun loadData() = Unit

}
